packages/contracts-bedrock/src/L1/SystemConfig.sol

```diff
@@ -19,11 +19,13 @@ contract SystemConfig is OwnableUpgradeable, ISemver {
     /// @custom:value GAS_LIMIT            Represents an update to gas limit on L2.
     /// @custom:value UNSAFE_BLOCK_SIGNER  Represents an update to the signer key for unsafe
     ///                                    block distrubution.
+    /// @custom:value CHALLENGER           Represents an update to the challenger address.
     enum UpdateType {
         BATCHER,
         GAS_CONFIG,
         GAS_LIMIT,
-        UNSAFE_BLOCK_SIGNER
+        UNSAFE_BLOCK_SIGNER,
+        CHALLENGER
```

Comment:
We don't need the challenger in this enum, it will break the `op-node` if an unknown type is emitted here

Reply:
OK, replaced with a dedicated event for challenger changes.